hub/__init__.py

```diff
@@ -10,12 +10,12 @@
     "tests": False,
 }
 
-from .api.dataset import Dataset
+from .api.dataset_handler import dataset
 from .api.tensor import Tensor
 from .api.load import load
 from .util.bugout_reporter import hub_reporter
 
-__all__ = ["Dataset", "Tensor", "load", "__version__"]
+__all__ = ["dataset", "Tensor", "load", "__version__"]
```

Comment:
i thought the decision was to expose all of these methods via `hub.{method}` as well as `hub.datasets.{method}`?

Reply:
Was it? I was following the api doc.

Reply:
yea it was, davit also just said that was it and he was against it

Reply:
https://www.notion.so/activeloop/Dataset-API-9d290f45f5b94b28b73ee7540be3ce8c the very first section

Reply:
aliases hub.empty for hub.dataset.empty